src/Schema/Directives/UpsertDirective.php

```diff
@@ -21,6 +21,12 @@ public static function definition(): string
   """
   model: String
 
+  """
+  Specify the columns by which to upsert the model.
+  This is optional, defaults to the ID or model Key.
```

Comment:
```suggestion
  Optional, by default `id` or the primary key of the model are used.
```

It is not like this argument itself defaults to those values, the mechanics behind it behave that way.